src/components/common/CheckboxGroup.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { useFormContext } from 'react-hook-form';
+
+interface CheckboxGroupProps {
+  name: string;
+  label: string;
+  options: { value: string; label: string }[];
+  required?: boolean;
+}
+
+const CheckboxGroup: React.FC<CheckboxGroupProps> = ({
```

Comment:
Please also check as above, I think it's more common to see the props after the args